src/theory/quantifiers/alpha_equivalence.cpp

```diff
@@ -23,9 +23,9 @@ using namespace CVC4::theory::quantifiers;
 using namespace CVC4::kind;
 
 struct sortTypeOrder {
```

Comment:
1) Put in anonymous namespace.
2) Replace with:
class SortTypeOrder {
 public:
  SortTypeOrder(TermUtil* tu) : d_tu(tu) { Assert(d_tu); }
  bool operator() (TypeNode i, TypeNode j) {
    return d_tu->getIdForType( i ) < d_tu->getIdForType( j );
  }
 private:
  TermUtil* d_tu;
};

Reply:
As I mentioned, I do not want to make changes to this commit beyond moving the code.  I am trying to keep the commits understandable.